src/pages/[...slug].tsx

```diff
@@ -149,139 +164,52 @@ const ContentPage: NextPage<ContentPiece> = ({
-                                    marginTop: ['inc20', null, null, '16px'],
-                                }}
-                            >
-                                Continuously Building and Hosting our Swift DocC
-                                Documentation using Github Actions and Netlify
-                            </TypographyScale>
-                            <Button
-                                sx={{ marginBottom: '40px' }}
-                                variant="secondary"
-                            >
-                                Continue
-                            </Button>
-                            <hr />
-                            <TypographyScale
-                                variant="heading6"
-                                sx={{
-                                    marginBottom: ['inc20', null, null, '16px'],
-                                    marginTop: ['inc20', null, null, '40px'],
-                                }}
-                            >
-                                More in this series
-                            </TypographyScale>
-                            <List
-                                glyph="circle"
-                                items={series}
-                                sx={{
-                                    marginBottom: ['inc20', null, null, '16px'],
-                                    marginTop: ['inc20', null, null, '16px'],
-                                }}
-                            />
-                        </Panel>
-                        <TypographyScale variant="heading5">
-                            Related
-                        </TypographyScale>
-                        {relatedContent.map(content => {
-                            return (
-                                <Card
-                                    sx={{ marginBottom: '26px' }}
-                                    key={content.title}
-                                    title={content.title}
-                                    description={content.description}
-                                    contentDate={content.contentDate}
-                                    pillCategory={content.category}
-                                    variant="small"
-                                    href={'/' + content.slug}
-                                />
-                            );
-                        })}
-                        <Button
-                            sx={{ marginBottom: '40px' }}
-                            variant="secondary"
-                        >
-                            Request a _____
-                        </Button> */}
-        </>
+                    ))}
+                </Grid>
+            </div>
+            <div sx={{ display: 'flex', justifyContent: 'center' }}>
+                <Button
+                    onClick={() => setRequestContentModalStage('text')}
```

Comment:
Assuming this is acting as a placeholder?

Reply:
No, `text` is a modal stage representing the stage where we prompt the user for text input.